packages/app/components/TabsHeader/TabItem.tsx

```diff
@@ -8,21 +8,23 @@ interface Props {
 	item: TabAble;
 	backgroundColor: string;
 	color: string;
+	fontWeight: '500' | '400';
```

Comment:
prefer leave it freedom as `string`, otherwise if we do want to control fontWeight make a `type` for it, e.g in `@walless/core`:
```typescript
export type FontWeights = '300' | '400' | '500' ....;
```